cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/OpenAI.scala

```diff
@@ -45,6 +45,45 @@ trait HasPrompt extends HasServiceParams {
   def setPromptCol(v: String): this.type = setVectorParam(prompt, v)
 }
 
+trait HasBulkPrompt extends HasServiceParams {
```

Comment:
Is this our naming or theirs? Perhaps something like `Batch` instead of `Bulk`

Reply:
They don't really have a set name. In the swagger they call it a list and elsewhere they call it an array.  I'll make the change to Batch.